account_payment_term_partner_holiday/models/account_move.py

```diff
@@ -41,4 +41,5 @@ def action_post(self):
         """
         for item in self:
             _item = item.with_context(move_partner_id=item.partner_id.id)
-            return super(AccountMove, _item).action_post()
+            res = super(AccountMove, _item).action_post()
+        return res
```

Comment:
This is confusing as we're overwriting the result value in every loop. I know the original method always returns `True`. Maybe that's what we should return no matter what.